cmake fails to configure if no Qt install is found

`blifexplorer` requires Qt5, however this seems like a very optional tool. Currently cmake fails if no qt is found;
```
CMake Warning at blifexplorer/CMakeLists.txt:9 (find_package):
  By not providing "FindQt5Widgets.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "Qt5Widgets", but CMake did not find one.

  Could not find a package configuration file provided by "Qt5Widgets" with
  any of the following names:

    Qt5WidgetsConfig.cmake
    qt5widgets-config.cmake

  Add the installation prefix of "Qt5Widgets" to CMAKE_PREFIX_PATH or set
  "Qt5Widgets_DIR" to a directory containing one of the above files.  If
  "Qt5Widgets" provides a separate development package or SDK, be sure it has
  been installed.


CMake Warning at blifexplorer/CMakeLists.txt:10 (find_package):
  By not providing "FindQt5Gui.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Qt5Gui", but
  CMake did not find one.

  Could not find a package configuration file provided by "Qt5Gui" with any
  of the following names:

    Qt5GuiConfig.cmake
    qt5gui-config.cmake

  Add the installation prefix of "Qt5Gui" to CMAKE_PREFIX_PATH or set
  "Qt5Gui_DIR" to a directory containing one of the above files.  If "Qt5Gui"
  provides a separate development package or SDK, be sure it has been
  installed.


CMake Warning at blifexplorer/CMakeLists.txt:11 (find_package):
  By not providing "FindQt5Core.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Qt5Core", but
  CMake did not find one.

  Could not find a package configuration file provided by "Qt5Core" with any
  of the following names:

    Qt5CoreConfig.cmake
    qt5core-config.cmake

  Add the installation prefix of "Qt5Core" to CMAKE_PREFIX_PATH or set
  "Qt5Core_DIR" to a directory containing one of the above files.  If
  "Qt5Core" provides a separate development package or SDK, be sure it has
  been installed.


CMake Error at blifexplorer/CMakeLists.txt:15 (message):
  Failed to find Qt5Widgets required for blifexplorer (on debian/ubuntu try
  'sudo apt install qt5-default')


-- Configuring incomplete, errors occurred!
See also "/home/kbuilder/vtr-build/CMakeFiles/CMakeOutput.log".
See also "/home/kbuilder/vtr-build/CMakeFiles/CMakeError.log".
Makefile:66: recipe for target 'all' failed
```

--- a/blifexplorer/CMakeLists.txt
+++ b/blifexplorer/CMakeLists.txt
@@ -12,60 +12,57 @@ find_package(Qt5Core)
 
 #Check dependencies
 if(NOT Qt5Widgets_FOUND)
-    message(FATAL_ERROR "Failed to find Qt5Widgets required for blifexplorer (on debian/ubuntu try 'sudo apt install qt5-default')")
+    message(STATUS "Failed to find Qt5Widgets required for blifexplorer (on debian/ubuntu try 'sudo apt install qt5-default')")
+elseif(NOT Qt5Gui_FOUND)
+    message(STATUS "Failed to find Qt5Gui required for blifexplorer (on debian/ubuntu try 'sudo apt install qt5-default')")
+elseif(NOT Qt5Core_FOUND)
+    message(STATUS "Failed to find Qt5Core required for blifexplorer (on debian/ubuntu try 'sudo apt install qt5-default')")
+else
+    set(CMAKE_AUTOMOC ON)
+    set(CMAKE_AUTOUIC ON)
+    set(CMAKE_AUTORCC ON)
+
+    #Collect the source files
+    set(QRC_EXPLORERRES src/explorerres.qrc)
+
+    set(UI_CLOCKCONFIG src/clockconfig.ui)
+
+    file(GLOB_RECURSE EXEC_SOURCES src/*.cpp)
+
+    file(GLOB_RECURSE EXEC_HEADERS src/*.h)
+    files_to_dirs(EXEC_HEADERS EXEC_INCLUDE_DIRS)
+
+    #The generated header files appear in the CMAKE_CURRENT_BINARY_DIR, so ensure
+    #it is in the include directories
+    include_directories(${EXEC_INCLUDE_DIRS} ${CMAKE_CURRENT_BINARY_DIR})
+
+    #Create the executable
+    add_executable(blifexplorer 
+                    ${UI_CLOCKCONFIG}
+                    ${QRC_EXPLORERRES}
+                    ${EXEC_SOURCES} 
+                    ${EXEC_HEADERS}
+    )
+
+    target_link_libraries(blifexplorer
+                            liblog
+                            libarchfpga
+                            libvtrutil
+                            libodin_ii
+                            libargparse
+                            Qt5::Widgets
+                            Qt5::Core
+                            Qt5::Gui
+                            ${CMAKE_DL_LIBS}
+    )
+
+    #Supress IPO link warnings if IPO is enabled
+    get_target_property(TEST_BLIFEXPLORER_USES_IPO blifexplorer INTERPROCEDURAL_OPTIMIZATION)
+    if (TEST_BLIFEXPLORER_USES_IPO)
+        set_target_properties(blifexplorer PROPERTIES LINK_FLAGS ${IPO_LINK_WARN_SUPRESS_FLAGS})
+    endif()
+
+
+    install(TARGETS blifexplorer DESTINATION bin)
+
 endif()
-
-if(NOT Qt5Gui_FOUND)
-    message(FATAL_ERROR "Failed to find Qt5Gui required for blifexplorer (on debian/ubuntu try 'sudo apt install qt5-default')")
-endif()
-
-if(NOT Qt5Core_FOUND)
-    message(FATAL_ERROR "Failed to find Qt5Core required for blifexplorer (on debian/ubuntu try 'sudo apt install qt5-default')")
-endif()
-
-set(CMAKE_AUTOMOC ON)
-set(CMAKE_AUTOUIC ON)
-set(CMAKE_AUTORCC ON)
-
-#Collect the source files
-set(QRC_EXPLORERRES src/explorerres.qrc)
-
-set(UI_CLOCKCONFIG src/clockconfig.ui)
-
-file(GLOB_RECURSE EXEC_SOURCES src/*.cpp)
-
-file(GLOB_RECURSE EXEC_HEADERS src/*.h)
-files_to_dirs(EXEC_HEADERS EXEC_INCLUDE_DIRS)
-
-#The generated header files appear in the CMAKE_CURRENT_BINARY_DIR, so ensure
-#it is in the include directories
-include_directories(${EXEC_INCLUDE_DIRS} ${CMAKE_CURRENT_BINARY_DIR})
-
-#Create the executable
-add_executable(blifexplorer 
-                ${UI_CLOCKCONFIG}
-                ${QRC_EXPLORERRES}
-                ${EXEC_SOURCES} 
-                ${EXEC_HEADERS}
-)
-
-target_link_libraries(blifexplorer
-                        liblog
-                        libarchfpga
-                        libvtrutil
-                        libodin_ii
-                        libargparse
-                        Qt5::Widgets
-                        Qt5::Core
-                        Qt5::Gui
-                        ${CMAKE_DL_LIBS}
-)
-
-#Supress IPO link warnings if IPO is enabled
-get_target_property(TEST_BLIFEXPLORER_USES_IPO blifexplorer INTERPROCEDURAL_OPTIMIZATION)
-if (TEST_BLIFEXPLORER_USES_IPO)
-    set_target_properties(blifexplorer PROPERTIES LINK_FLAGS ${IPO_LINK_WARN_SUPRESS_FLAGS})
-endif()
-
-
-install(TARGETS blifexplorer DESTINATION bin)

